### PR TITLE
[16.0][MIG] partner_company_group: Migration to 16.0

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@luisg123v Please review

@desdelinux

File: partner_company_group/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Partner Company Group",
+    "summary": "Adds the possibility to add a company group to a company",
+    "version": "16.0.1.0.0",
+    "category": "Sales",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": [
```
Comment (luisg123v) [inline partner_company_group/__manifest__.py:11]: Could you add myself as maintainer, please?

Just add the following line above this one:

`"maintainers": ["luisg123v"],`

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@luisg123v Please review

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

@moylop260 could you review/merge, please?

CC @desdelinux

**Pull request merged**